tests/sles4sap/sap_deployment_automation_framework/cleanup.pm

```diff
@@ -13,26 +13,42 @@ use parent 'sles4sap::sap_deployment_automation_framework::basetest';
 use strict;
 use testapi;
 use warnings;
-use sles4sap::sap_deployment_automation_framework::deployment;
-use sles4sap::console_redirection;
+use sles4sap::sap_deployment_automation_framework::deployment
+  qw(serial_console_diag_banner
+  sdaf_cleanup
+  az_login
+  load_os_env_variables);
+use sles4sap::sap_deployment_automation_framework::deployment_connector qw(find_deployer_resources);
+use sles4sap::console_redirection qw(connect_target_to_serial disconnect_target_from_serial);
+use sles4sap::azure_cli qw(az_resource_delete);
 
 sub test_flags {
     return {fatal => 1};
 }
 
 sub run {
-    serial_console_diag_banner('end: sdaf_cleanup.pm');
+    serial_console_diag_banner('Start: sdaf_cleanup.pm');
     if (get_var('SDAF_RETAIN_DEPLOYMENT')) {
         record_info('Cleanup OFF', 'OpenQA variable "SDAF_RETAIN_DEPLOYMENT" is active, skipping cleanup.');
         return;
     }
 
-    # Cleanup SDAF files form Deployer VM
+    # Trigger SDAF remover script and clean up files from deployer VM
     connect_target_to_serial();
     load_os_env_variables();
     az_login();
     sdaf_cleanup();
     disconnect_target_from_serial();
+
+    # Cleanup deployer VM resources
+    my @resource_cleanup_list = @{find_deployer_resources(return_ids => 1)};
```

Comment:
`id` is the default for `return_ids` .  Here, even if ti is default,  you explicitly set it to 1 (I like explicit) but in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19654/files#diff-a14cb46e70564f2332ed9893b6322589269b38f6d34cd078045b3ff5df61c717R37 you are not

Reply:
It was the other way around. 
`$args{return_ids} = $args{return_ids} ? 'id' : 'name';` 
if `$args{return_ids} ` was undefined you would get resource name by default. 

With your suggestion I just changed it to
`$args{return_value} //= 'name';` 
Thanks... it is a lot less confusing that way.